src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkContentTests.cs

```diff
@@ -47,31 +210,43 @@ private void WriteTarballFileList(string? tarballPath, string outputFileName, bo
         fileListing = BaselineHelper.RemoveRids(fileListing, isPortable);
         fileListing = BaselineHelper.RemoveVersions(fileListing);
         IEnumerable<string> files = fileListing.Split(Environment.NewLine).OrderBy(path => path);
-        files = RemoveExclusions(
-                    files,
-                    GetExclusionFilters(
-                        Path.Combine(BaselineHelper.GetAssetsDirectory(), "SdkDiffExclusions.txt"),
-                        sdkType));
+        files = RemoveExclusions(files, GetSdkDiffExclusionFilters(sdkType));
 
         File.WriteAllLines(outputFileName, files);
     }
 
-        private static IEnumerable<string> RemoveExclusions(IEnumerable<string> files, IEnumerable<string> exclusions) =>
-            files.Where(item => !exclusions.Any(p => FileSystemName.MatchesSimpleExpression(p, item)));
+    private static IEnumerable<string> RemoveExclusions(IEnumerable<string> files, IEnumerable<string> exclusions) =>
+        files.Where(item => !IsFileExcluded(item, exclusions));
 
-        private static IEnumerable<string> GetExclusionFilters(string exclusionsFilePath, string sdkType)
-        {
-            int prefixSkip = sdkType.Length + 1;
-            return File.ReadAllLines(exclusionsFilePath)
-                .Where(line => line.StartsWith(sdkType + ",")) // process only specific sdk exclusions
-                .Select(line =>
-                {
-                    // Ignore comments
-                    var index = line.IndexOf('#');
-                    return index >= 0 ? line[prefixSkip..index].TrimEnd() : line[prefixSkip..];
-                })
-                .ToList();
-        }
+    private static bool IsFileExcluded(string filePath, IEnumerable<string> exclusions) =>
+        exclusions.Any(p => FileSystemName.MatchesSimpleExpression(p, filePath));
+
+    private static IEnumerable<string> GetSdkDiffExclusionFilters(string sdkType)
+    {
+        string exclusionsFilePath = Path.Combine(BaselineHelper.GetAssetsDirectory(), "SdkFileDiffExclusions.txt");
```

Comment:
Would it make sense for ParseExclusionsFile to take in a exclusionsFileName and have it formulate the path?  It looks like  calls to the method do this?  This would allow this method to be declared as an expression bodied method.